### PR TITLE
:memo: Add readme

### PR DESCRIPTION
**Pull request by martinfoldyna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+ # Ackee fela
+ 
+Set of commonly used hooks and types
+---
+## Installation
+`$ yarn add @ackee/fela`
```
Comment (horaklukas) [inline README.md:6]: better to use a block code with syntax higlighting

```shell
...
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+export type Theme = typeof theme;
+
+// index.d.ts 
+import { Theme as ProjectTheme } from "./theme.ts"
+
+declare module "@ackee/fela" {
+  interface Theme extends ProjectTheme {}
+}
+```
```
Comment (horaklukas) [inline README.md:149]: the section about theme extending is quite lost deep down in the documentation, although it's quite essential and something you wish to do on every project with theme. Consider relocating it upper with proper title like "Using own theme" or something like that...

**Review by horaklukas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (martinfoldyna) [reply README.md]: Done ✅

File: README.md (hunk 2 above)
Comment (martinfoldyna) [reply README.md]: Okay, I changed it a little. Do you think it is better like this? 🙂

Comment (horaklukas) [reply README.md]: definitely better 👍

**Pull request merged**